go.mod

```diff
@@ -1,15 +1,23 @@
 module github.com/aws/aws-xray-daemon
 
-go 1.16
+go 1.18
```

Comment:
In order to affect the version of Golang used in the Docker images, we need to update the version of the Golang base image in the Dockerfiles:
1. https://github.com/aws/aws-xray-daemon/blob/master/Dockerfile.amazonlinux#L2
2. https://github.com/aws/aws-xray-daemon/blob/master/Dockerfile#L2

And we should bump the version of Go we use in the CI/CD too probably, e.g. https://github.com/aws/aws-xray-daemon/blob/dd6aef5fc1b15ca4afecc93b491fd267d3c4eded/.github/workflows/release-build-publish.yml#L25

Reply:
Good catch. That was off my radar.